tests/common_testing/alfasim_sdk_common_testing/filled_case_descriptions.py

```diff
@@ -359,6 +359,13 @@
     well_max_flow_rate=Scalar(1.0, "sm3/d"),
     well_max_flow_rate_curve=Curve(Array([1.0, 1.0], "sm3/d"), Array([0, 1], "s")),
 )
+UNDERSATURATED_OIL_IPR_DESCRIPTION = case_description.UndersaturatedOilIPRDescription(
+    well_index_phase=constants.WellIndexPhaseType.Oil,
+    min_pressure_difference=Scalar(0.0, "bar"),
+    bubble_point_pressure=Scalar(0.0, "bar"),
+    well_index=Scalar(1.0e-6, "m3/Pa.s"),
+    well_index_curve=Curve(Array([1.0e-6, 0.9e-6], "m3/Pa.s"), Array([0, 1], "h")),
+)
```

Comment:
Those should be values diferent from the default values to check if the conversion "case description"<->"alfacase" round trip is no losing any data.

Reply:
Looks like Vogel values are the default ones too, right? If so, they could be fixed here.

Reply:
Thanks, both are done!